upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.16.yaml.template

```diff
@@ -3857,8 +3861,17 @@ spec:
             # The default IPv4 pool to create on startup if none exists. Pod IPs will be
             # chosen from this range. Changing this value after installation will have
             # no effect. This should fall within `--cluster-cidr`.
+            {{- if (WithDefaultBool .Networking.Calico.IPv6Support false) }}
+            - name: CALICO_IPV6POOL_CIDR
+              value: "{{ .KubeControllerManager.ClusterCIDR }}"
```

Comment:
According to https://kubernetes.io/docs/concepts/services-networking/dual-stack/ the KCM `--cluster-cidr` is, in dual-stack, an IPv4 CIDR and an IPv6 CIDR, separated by a comma. It doesn't look like calico/node accepts that syntax in either CALICO_IPV6POOL_CIDR or CALICO_IPV4POOL_CIDR.

I think we'd need a template function to parse `ClusterCIDR` into its two components.

Reply:
Actually, I'm okay with not supporting dual-stack.

Reply:
For now, single stack ClusterCIDR is ok from my point of view too.
Dual-stack is also blocked by https://github.com/kubernetes/cloud-provider-aws/pull/230, as the kubelet expects the cloud provider to return both IPv4 and IPv6 for the node.